tests/p2p/netfilter/test_utils.py

```diff
@@ -0,0 +1,26 @@
+from hathor.p2p.netfilter import get_table
+from hathor.p2p.netfilter.utils import add_peer_id_blacklist
+from tests import unittest
+
+
+class NetfilterUtilsTest(unittest.TestCase):
```

Comment:
I feel we also need to test the comparison of objects.

Reply:
The comparison methods were removed and the tests were improved